ports/raspberrypi/boards/lilygo_t_display_rp2040/board.c

```diff
@@ -0,0 +1,153 @@
+    0xC3, 1, 0x10,
+    0xC4, 1, 0x20,
+    0xC6, 1, 0x0F,
+    0xD0, 2, 0xA4, 0xA1,
+    // ST7789V gamma setting
+    0xE0, 14, 0xD0, 0x00, 0x02, 0x07, 0x0A, 0x28, 0x32, 0x44, 0x42, 0x06, 0x0E, 0x12, 0x14, 0x17,
+    0xE1, 14, 0xD0, 0x00, 0x02, 0x07, 0x0A, 0x28, 0x31, 0x54, 0x47, 0x0E, 0x1C, 0x17, 0x1B, 0x1E,
+    0x21, 0,
+    // display on
+    0x29, 0 | DELAY, 255,
+};
+
+static void display_init(void) {
+    busio_spi_obj_t *spi = &displays[0].fourwire_bus.inline_bus;
+
+    common_hal_busio_spi_construct(
+        spi,
+        &pin_GPIO2,     // CLK
+        &pin_GPIO3,     // MOSI
+        NULL,           // MISO not connected
+        false);         // Not half-duplex
+
+    common_hal_busio_spi_never_reset(spi);
+
+    displayio_fourwire_obj_t *bus = &displays[0].fourwire_bus;
+    bus->base.type = &displayio_fourwire_type;
+
+    common_hal_displayio_fourwire_construct(
+        bus,
+        spi,
+        &pin_GPIO1,     // DC
+        &pin_GPIO5,     // CS
+        NULL,           // RST (Reset pin tie to 0, do not set here)
+        40000000,       // baudrate
+        1,              // polarity
+        0               // phase
+        );
+
+    displayio_display_obj_t *display = &displays[0].display;
+    display->base.type = &displayio_display_type;
+
+    common_hal_displayio_display_construct(
+        display,
+        bus,
+        240,            // width (after rotation)
+        135,            // height (after rotation)
+        52,             // column start
+        40,             // row start
+        90,             // rotation
+        16,             // color depth
+        false,          // grayscale
+        false,          // pixels in a byte share a row. Only valid for depths < 8
+        1,              // bytes per cell. Only valid for depths < 8
+        false,          // reverse_pixels_in_byte. Only valid for depths < 8
+        true,           // reverse_pixels_in_word
+        MIPI_COMMAND_SET_COLUMN_ADDRESS, // set column command
+        MIPI_COMMAND_SET_PAGE_ADDRESS,   // set row command
+        MIPI_COMMAND_WRITE_MEMORY_START, // write memory command
+        display_init_sequence,
+        sizeof(display_init_sequence),
+        &pin_GPIO4,     // backlight pin
+        NO_BRIGHTNESS_COMMAND,
+        1.0f,           // brightness (ignored)
+        false,          // auto_brightness
+        false,          // single_byte_bounds
+        false,          // data_as_commands
+        true,           // auto_refresh
+        60,             // native_frames_per_second
+        true,           // backlight_on_high
+        false           // SH1107_addressing
+        );
+
+    common_hal_never_reset_pin(&pin_GPIO4); // backlight pin
+}
+
+void board_init(void) {
+    // Pin 22 has to be pulled high to turn on the LCD
+    const uint PWR_PIN = LCD_POWER;
+    gpio_init(PWR_PIN);
+    gpio_set_dir(PWR_PIN, GPIO_OUT);
+    gpio_put(PWR_PIN, 1);
+    common_hal_never_reset_pin(&pin_GPIO22);
```

Comment:
Seems not to work with my board as soon as I use REPL with the regular build and ST7789 library. Compiling these 5 file changes into a circuitpython build beta 8.1.0 shows only during boot for a short period the display content (backlight is on all the time) but is otherwise dark.